OOModel/src/expressions/MetaCallExpression.h

```diff
@@ -49,6 +52,22 @@ class OOMODEL_API MetaCallExpression: public Super<Expression>
 		 * Returns the meta definition used in this meta call if one exists.
 		 */
 		MetaDefinition* metaDefinition();
+
+		/**
+		 * Generates the tree resulting from this meta call and returns it.
+		 */
+		Declaration* generate();
```

Comment:
The current separation originates from me not wanting the visualization to perform the actual generation when it asks for the generated tree. If we do not care about that I can change it back to just one "generate" method.